packages/auth-providers/supabase/api/tsconfig.json

```diff
@@ -3,8 +3,10 @@
   "compilerOptions": {
     "strict": true,
     "rootDir": "src",
-    "outDir": "dist"
+    "outDir": "dist",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext"
```

Comment:
I can't see why we need these changes in the api package! Maybe I'm missing something